### PR TITLE
Clean up links in ReadTheDocs

### PR DESCRIPTION
**Pull request by TrimBiggs**

### CODE REVIEW

File: docs/source/contribute.rst [hunk 1]
```diff
@@ -37,8 +37,8 @@ Features or any changes to the codebase should be done as follows:
 5. Reviewer process:
 
    -  Receive notice of review by GitHub email, GitHub notification, or by
-      checking `all your Project Calico GitHub
-      issues <https://github.com/organizations/projectcalico/dashboard/issues/assigned?direction=desc&state=open>`__.
+      checking your assigned `Project Calico GitHub
+      issues <https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+user%3Aprojectcalico>`__.
```
Comment (robbrockbank) [inline docs/source/contribute.rst]: that's a bit of an ugly URL to display.  perhaps we should just hyperlink it.

Comment (TrimBiggs) [inline docs/source/contribute.rst]: This is a hyperlink in ReadTheDocs

**Comment by robbrockbank** [comment]
One suggestion and then LGTM.

File: docs/source/contribute.rst (hunk 1 above)
Comment (robbrockbank) [inline docs/source/contribute.rst]: Ah apologies - I was in some odd MD/RTD hybrid mode.

**Comment by eepyaich** [comment]
These all look good to me - merging.

**Pull request merged**